packages/cli/src/commands/create/CreateCliApp.ts

```diff
@@ -70,6 +74,7 @@ export const createAppCommand = async ({
   dryRun,
   publisherMintAddress,
   storageParams,
+  priorityFeeLamports = 500000,
```

Comment:
Can we pull this constant out somewhere, and import it where used? That'll make any future updates trivial.

Reply:
Done.